ajax/ajaxVote.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+include('config/init.php');
+
```

Comment:
For pages like this that only work for logged in users, I'd check that they're logged in before doing anything else